core/sail/shacl/src/test/resources/test-cases/hasValueIn/or/shacl.ttl

```diff
@@ -0,0 +1,16 @@
+@base <http://example.com/ns> .
+@prefix ex: <http://example.com/ns#> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix dash: <http://datashapes.org/dash#>.
+
+ex:shape1
+	a sh:NodeShape ;
+	sh:targetClass ex:Person ;
+	sh:property [
+		sh:path ex:info ;
+		sh:or ([ dash:hasValueIn ("blue") ; ] [ dash:hasValueIn ("green") ; ] [ dash:hasValueIn ("red"); ]) ;
```

Comment:
I don't object to having a test for a corner case, but isn't this an odd use of `dash:hasValueIn`? I thought the point was to allow use of a list so you don't have to use an or constraint at all. In other words, isn't the above equivalent to just saying:

     sh:path ex:info;
     dash:hasValueIn ("blue" "green" "red");

Reply:
That would be equivalent to:
```
sh:or ([sh:path ex:info ; dash:hasValueIn ("blue") ; ] [sh:path ex:info ; dash:hasValueIn ("green") ; ] [sh:path ex:info ; dash:hasValueIn ("red"); ]) ;
```

Reply:
Ah I see! I think I'm beginning to understand what the actual issue here is. At least, I think so...